homeassistant/components/jewish_calendar/config_flow.py

```diff
@@ -0,0 +1,158 @@
+                if all(
+                    _value == entry.as_dict().get(_key)
+                    for _key, _value in user_input.items()
+                    if _key != CONF_NAME
+                ):
+                    return self.async_abort(reason="already_configured")
+
+                if entry.data[CONF_NAME] == user_input[CONF_NAME]:
+                    errors[CONF_NAME] = "name_exists"
+                    break
+
+            try:
+                info = await validate_input(self.hass, user_input)
+                return self.async_create_entry(title=info["title"], data=user_input)
+            except Exception:  # pylint: disable=broad-except
+                _LOGGER.exception("Unexpected exception")
+                errors["base"] = "unknown"
+
+        return self.async_show_form(
+            step_id="user",
+            data_schema=vol.Schema(
+                {
+                    vol.Optional(CONF_NAME, default=DEFAULT_NAME): str,
+                    vol.Required(
+                        CONF_DIASPORA, default=DEFAULT_DIASPORA
+                    ): BooleanSelector(),
+                    vol.Required(
+                        CONF_LANGUAGE, default=DEFAULT_LANGUAGE
+                    ): SelectSelector(SelectSelectorConfig(options=LANGUAGE)),
+                    vol.Required(
+                        CONF_LOCATION,
+                        default={
+                            "latitude": self.hass.config.latitude,
+                            "longitude": self.hass.config.longitude,
+                        },
+                    ): LocationSelector(),
+                    vol.Optional(
+                        CONF_ELEVATION, default=self.hass.config.elevation
+                    ): int,
+                    vol.Optional(
+                        CONF_TIME_ZONE, default=self.hass.config.time_zone
+                    ): SelectSelector(
+                        SelectSelectorConfig(
+                            options=sorted(zoneinfo.available_timezones()),
+                        )
+                    ),
+                }
+            ),
+            errors=errors,
+        )
+
+    async def async_step_import(self, import_config: ConfigType | None) -> FlowResult:
+        """Import a config entry from configuration.yaml."""
+        return await self.async_step_user(import_config)
+
+
+class JewishCalendarOptionsFlowHandler(OptionsFlow):
+    """Handle Jewish Calendar options."""
+
+    def __init__(self, config_entry: ConfigEntry) -> None:
+        """Initialize Jewish Calendar options flow."""
+        self.config_entry = config_entry
+
+    async def async_step_init(
+        self, user_input: dict[str, str] | None = None
+    ) -> FlowResult:
+        """Manage the Jewish Calendar options."""
+        if user_input is not None:
+            return self.async_create_entry(title="", data=user_input)
+
+        options = {
+            vol.Optional(
```

Comment:
You should make the OPTIONS_SCHEMA a constant at the top of the file, and then leverage `add_suggested_values` to inject existing options.